.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -26,9 +26,6 @@ parameters:
   type: boolean
   default: True
 
```

Comment:
could we add a comment for how we set variables via variable group name?

Reply:
Thanks for the comment. Added it.